client/src/styles/leafletComponents/controlFrame.less

```diff
@@ -8,7 +8,7 @@
 .s-container {
   width      : 100%;
   background : white;
-  padding    : 5px;
```

Comment:
This is used in several places so will cascade.  Instead, I would add the change where you need it